packages/core/src/components/reviews/ReviewModal/ReviewModal.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Button,
+  type ModalProps as UIModalProps,
+  type ModalHeaderProps as UIModalHeaderProps,
+  type ModalBodyProps as UIModalBodyProps,
+  type ModalFooterProps as UIModalFooterProps,
+  useUI,
+} from '@faststore/ui'
+
+import styles from './section.module.scss'
+
+import dynamic from 'next/dynamic'
+
+const UIModal = dynamic<UIModalProps>(
+  () =>
+    import(/* webpackChunkName: "UIModal" */ '@faststore/ui').then(
+      (mod) => mod.Modal
```

Comment:
```suggestion
      (module) => module.Modal
```
nitpick: just to follow the previous pattern.